cql-engine-fhir/src/main/java/org/opencds/cqf/cql/terminology/fhir/FhirTerminologyProvider.java

```diff
@@ -73,6 +73,21 @@ public FhirTerminologyProvider withBasicAuth(String userName, String password) {
     public boolean in(Code code, ValueSetInfo valueSet) throws ResourceNotFoundException {
         // Potential problems:
         // ValueSetInfo void of id --> want .ontype() instead
+        try {
+            URL url = new URL(valueSet.getId());
+            Bundle searchResults = fhirClient.search().forResource(ValueSet.class).where(ValueSet.URL.matches().value(url.toString())).returnBundle(Bundle.class).execute();
+            if (searchResults.hasEntry()) {
+                if (searchResults.getEntryFirstRep().hasResource()) {
```

Comment:
On the searchResults, if there is no value set for this URL, should that be an exception?